app/uk/gov/hmrc/nisp/views/account.scala.html

```diff
@@ -35,7 +34,10 @@
     isPertaxUrl:Boolean
 )(implicit request: Request[_], user: NispUser, partialRetriever: CachedStaticHtmlPartialRetriever)
 
-
+@pageScripts = {
```

Comment:
I think the NI record js just needs to be on the NI record page.
Since the breadcrumb needs to be everywhere I think that it should go in main.scala.html, just in case people forget to add it.